health/health.d/memory.conf

```diff
@@ -13,7 +13,10 @@ component: Memory
     every: 1m
      warn: $this > 0
     delay: down 1h multiplier 1.5 max 1h
-     info: number of ECC correctable errors in the last 10 minutes
+     info: Number of ECC correctable errors in the last 10 minutes. \
+           There are ECC correctable errors. \
+           This event does not immediately lead to problems, but it may indicate one of the DIMM modules is slowly failing. \
```

Comment:
It may also just be a transient error due to cosmic rays.

Reply:
We took this info from [Intel's System EventLog Troubleshooting Guide](https://www.intel.com/content/dam/support/us/en/documents/server-products/SystemEventLog_TroubleshootingGuide.pdf).

<img width="751" alt="Screenshot 2021-09-15 at 12 52 24" src="https://user-images.githubusercontent.com/22274335/133412170-5757cbba-6380-4635-9f82-0b89b07f6a2c.png">

---

What do you suggest?

Reply:
> It may also just be a transient error due to cosmic rays.

Nice to know, but I think it is ok to not mention it (unless NASA uses Netdata).

Reply:
I'll just leave this here... https://youtu.be/AaZ_RSt0KP8

Reply:
> Studies by IBM in the 1990s suggest that computers typically experience about one cosmic-ray-induced error per 256 megabytes of RAM per month.

Therefore, these errors should be correctable and very rare under normal conditions.

Reply:
[More information](https://stackoverflow.com/a/23587649/11451973) on the matter.

Reply:
OK, I probably should have stated my point a bit better, but _if_ this is a one-off error that never recurs, it’s _probably_ not an issue because it’s statistically most likely in that case to be a transient error (possibly cosmic ray induced, possibly induced by gamma radiation from elevated radon levels in the area, possibly just random interference from some other source). When it’s an issue is if it’s happening regularly